app/src/main/java/org/fossasia/susi/ai/helper/DateTimeHelper.kt

```diff
@@ -71,7 +71,11 @@ object DateTimeHelper {
         val sdf = SimpleDateFormat(" MMM dd, yyyy")
         val tz = TimeZone.getDefault()
         sdf.timeZone = tz
-        return sdf.format(formatDate(date))
+        if (formatDate(date) == null || sdf.format(formatDate(date)) == null) {
+            return ""
+        } else {
+            return sdf.format(formatDate(date))
+        }
```

Comment:
If this is needed, then the argument should be made non nullable

Reply:
Made date non nullable so this argument isn't needed any more, returning today date when cannot parse ie when null. please recheck...

Reply:
Then remove this if else

Reply:
removed this, and updated DateTimeHelperTest accordingly